test/test_serialization.py

```diff
@@ -585,8 +586,8 @@ def __exit__(self, *args, **kwargs):
 
 class TestBothSerialization(TestCase, SerializationMixin):
```

Comment:
This will run the tests in the mixin an additional time with the same arguments since they're now instantiated on CUDA and CPU:

test_load_error_msg (__main__.TestBothSerializationCPU) ... ok (0.102s)
test_load_error_msg (__main__.TestBothSerializationCUDA) ... ok (0.045s)

These tests run so quickly it's OK, but I think it'd also be fine to remove the mixin here. The next two test classes run the mixin tests with _use_new_zipfile_serialization=False/True, anyway, so this is just testing its default value (redundantly, since the default value is True).